source/clear-linux/guides/maintenance/swupd-mixer-integration.rst

```diff
@@ -0,0 +1,117 @@
+.. _swupd_mixer_integration:
+
+Adding custom packages and bundles
+##################################
+
+
+
+Prerequisites
+=============
+
+To use the new swupd mixer feature, you'll need a recent image of Clear
```

Comment:
Avoid contractions. They can be ambiguous and are not easy to understand for many non-native English readers. I'm only commenting this instance but all contractions should be replaced.